src/App.tsx

```diff
@@ -35,61 +35,122 @@ export function getReorderedGoods(
   const visibleGoods = [...goods];
 
   // Sort and reverse goods if needed
+  visibleGoods.sort((a, b) => {
+    switch (sortType) {
+      case SortType.ALPHABET: {
+        return a.localeCompare(b);
+      }
+
+      case SortType.LENGTH: {
+        return a.length - b.length;
+      }
+
+      case SortType.NONE: {
+        return 0;
+      }
+
+      default: {
+        throw new Error('Wrong sorting type!');
+      }
```

Comment:
you can just do it like this:
```suggestion
      default:
        return SortType.NONE;
```